_article-types/howto.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: article-type
+name: howto
```

Comment:
I think that might only for for permalinks. If I remove the `name` from the tools, for example, `{{ tool.name }}` doesn't give me anything